### PR TITLE
Gcp

### PR DESCRIPTION
**Pull request by sbneto**

### CODE REVIEW

File: s3conf/storages.py [hunk 1]
```diff
@@ -6,6 +6,7 @@
 
 import boto3
 from botocore.exceptions import ClientError
+from google.cloud import storage
```
Comment (alanjds) [inline s3conf/storages.py]: Maybe it is time to provide extras for this package. Is not nice to force people to install Google to use AWS or to install AWS to use Google.

Comment (sbneto) [reply s3conf/storages.py]: Completely agree, will do that in another PR though :+1:

File: s3conf/client.py [hunk 2]
```diff
@@ -53,7 +47,8 @@ def main(ctx, edit, create):
             if ctx.invoked_subcommand is None:
                 settings = config.Settings()
                 logger.debug('Using config file %s', settings.config_file)
-                STORAGES['local'](settings=settings).open(settings.config_file).edit(create=create)
+                StorageCls = storages.get_storage(settings.config_file)
+                StorageCls(settings=settings).open(settings.config_file).edit(create=create)
```
[the remark this answers is not in the document]
Comment (sbneto) [reply s3conf/client.py]: This will disappear in the next refactor. Big things are coming. :grimacing:

**Comment by sbneto** [comment]
As a matter of fact, I did not find a reasonable way to test the GCPStorage locally. People have discussed this on https://github.com/googleapis/google-cloud-python/issues/4897 and [StackOverflow Question](https://stackoverflow.com/questions/37542530/is-there-google-cloud-storage-emulator). There seems to be a way for [Java](https://github.com/googleapis/google-cloud-java/blob/master/TESTING.md#testing-code-that-uses-storage), but I could not find a simple solution for Python.

**Comment by alanjds** [comment]
> As a matter of fact, I did not find a reasonable way to test the GCPStorage locally. People have discussed this on [googleapis/google-cloud-python#4897](https://github.com/googleapis/google-cloud-python/issues/4897) and [StackOverflow Question](https://stackoverflow.com/questions/37542530/is-there-google-cloud-storage-emulator). There seems to be a way for [Java](https://github.com/googleapis/google-cloud-java/blob/master/TESTING.md#testing-code-that-uses-storage), but I could not find a simple solution for Python.

I am using `pytest-vcr` and marking the tests that do external calls. Well, the 1st run will be an integration test, but the next ones will be mocked using the recorded cassetes.

See: https://pypi.org/project/pytest-vcr/

**Pull request merged**